src/main/java/duke/system/Ui.java

```diff
@@ -0,0 +1,26 @@
+package duke.system;
+
+/**
+ * Represents the object that is responsible for displaying context to the user
+ * such as separating line or error
+ */
+public class Ui {
+    public void showLoadingError() {
+        System.out.println("Error in loading the list");
+    }
+
+    /**
+     * Print the input in the console
+     */
+    public void showText(String input) {
+        System.out.println(input);
+    }
+
+    public void showError(String input) {
+        System.out.println("Error: " + input);
```

Comment:
Just smt to consider: showError may confuse ppl a little since its usually associated with printing to stderr, but I guess its not a big deal at the same time 🤷